### PR TITLE
1.2 docs

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: _includes/manuals/1.2/3.5.1_initial_setup.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
-Foreman configuration is managed from two places; a configuration file *config/settings.yaml* and from the *SETTINGS/Foreman Settings* page. A full description of the configuration options is given at [foreman_configuration](#3.5.2ConfigurationOptions)
+Foreman configuration is managed from two places; a configuration file
+*config/settings.yaml* and from the *SETTINGS/Foreman Settings* page. A full
+description of the configuration options is given at
+[foreman_configuration](manuals/1.2/index.html#3.5.2ConfigurationOptions)
 
 #### Database
 
-Foreman uses a database, this database can be shared with Puppet store-configs (they are compatible, as Foreman extends the puppet database schema). By default, SQLite is used, if you want to use other database (e.g. [MySQL](#3.5.3DatabaseSetup)) please modify the configuration file under *config/database.yml*.
+Foreman uses a database, this database can be shared with Puppet store-configs
```
Comment (GregSutcliffe) [inline _includes/manuals/1.2/3.5.1_initial_setup.md:11]: This section about shared db stuff should be enitrely removed for 1.2, and optionally a block added about the deprecation of shared dbs linking to the prod2dev instructions.

File: _includes/manuals/1.2/3.5.1_initial_setup.md [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+(they are compatible, as Foreman extends the puppet database schema). By
+default, SQLite is used, if you want to use other database (e.g.
```
Comment (GregSutcliffe) [inline _includes/manuals/1.2/3.5.1_initial_setup.md:13]: SQlite is only the default by package installer, the installer uses Psql - perhaps we should state the difference?

Comment (domcleal) [inline _includes/manuals/1.2/3.5.1_initial_setup.md:13]: Let's do another pass to change these, I think these only came up in this PR as I was reapplying one of the other commits to the 1.2 manual.  I haven't been editing it myself!

**Comment by GregSutcliffe** [comment]
Happy to submit a PR of my own for the docs updates. One minor comment on 83880f4 otherwise :+1:

**Pull request closed without merging**